package.json

```diff
@@ -86,6 +86,7 @@
     "postcss-loader": "3.0.0",
     "progress-bar-webpack-plugin": "1.12.1",
     "promptly": "3.0.3",
+    "prop-types-elementtype": "^1.0.0",
```

Comment:
Doesn't look like this is needed? https://github.com/facebook/prop-types/pull/211 - from the text on the `prop-types-elementtype repo README.md:

> You can use this library until this PR is merged in the official PropTypes.

It looks like the prop-types library has `PropTypes.element` and `PropTypes.elementType` [as options](https://github.com/facebook/prop-types).

Reply:
Weird, I must have had an old version of prop-types installed locally because it wasn't working for me before.

Should be fixed now: 850dc6280e06a00a1e8ef65ac10b169ccdf504cc.